.claude/commands/textlint.md

```
@@ -0,0 +1,83 @@
+   ```bash
+   cp frontend/.claude/.textlintrc.json ~/.claude/.textlintrc.json
+   ```
+
+3. **Install required textlint plugins:**
+
+   ```bash
+   npm install -g @textlint/markdown preset-ja-technical-writing @textlint-ja/preset-ai-writing
+   ```
+
+4. **Test textlint configuration:**
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)